README.md

```diff
@@ -45,35 +45,37 @@ evaluate_dataset.to_parquet('your/path/to/evaluate_dataset.parquet')
 
 ### Evaluate your data to various RAG modules
 ```python
-from autorag import Evaluator
+from autorag.evaluator import Evaluator
 
-evaluator = Evaluator(qa_path='your/path/to/qa.parquet', corpus_path='your/path/to/corpus.parquet')
+evaluator = Evaluator(qa_data_path='your/path/to/qa.parquet', corpus_data_path='your/path/to/corpus.parquet')
 evaluator.start_trial('your/path/to/config.yaml')
 ```
 or you can use command line interface
 ```bash
-autorag evaluate --config your/path/to/default_config.yaml
+autorag evaluate --config your/path/to/default_config.yaml --qa_path your/path/to/qa.parquet --corpus_path your/path/to/corpus.parquet
```

Comment:
how about unify variable name of xxx_data_path and xxx_path
+ in this version, qa_path variable called by cli variable?